java/code/src/com/redhat/rhn/frontend/xmlrpc/proxy/ProxyHandler.java

```diff
@@ -274,16 +276,17 @@ public int activateProxy(String clientcert, String version)
      * @xmlrpc.param #param("string", "proxyKey", "proxy SSL private key in PEM format")
      *  @xmlrpc.returntype #array_single("byte", "binary object - package file")
      */
-    public byte[] containerConfig(User loggedInUser, String proxyName, String server, Integer maxCache, String email,
+    public byte[] containerConfig(User loggedInUser, String proxyName, Integer proxyPort, String server,
```

Comment:
should we add an overloaded method, keeping the old behavior and the user the old port as default?
If we just change the signature we will be a breaking change for this XML-RPC API method.

Reply:
well, it's breaking what has been released in Beta2... I wouldn't worry too much